spec/measure_repository_service_test_kit/library_group_spec.rb

```diff
@@ -88,4 +88,389 @@
       expect(result.result).to eq('fail')
     end
   end
+
+  describe 'Server successfully searches and retrieves specified resource by its url' do
```

Comment:
I would just change `specified resource` here and in the rest of the spec files to `Library`. Same in `Measure` file